src/SrBot.ts

```diff
@@ -222,12 +222,12 @@ export class SrBot {
      * Checks if a user is an admin.
      * @param user Username to check
      */
-    private isServerAdmin(serverId: string, user: string): boolean {
+    private isServerAdmin(serverId: string, user: string, permission: Discord.Permissions): boolean {
         const server = this.statsGenerator.getServer(serverId);
         if (!server){
             return false;
         }
-        return server.admins.map(user => user.toUpperCase()).includes(user.toUpperCase());
+        return server.admins.map(user => user.toUpperCase()).includes(user.toUpperCase()) || permission.has('ADMINISTRATOR') || permission.has('MANAGE_GUILD');
```

Comment:
Do you want to remove `.includes(user.toUpperCase()` in case of username switching?  If so, remove it from overwatch.json as well

Reply:
Also what is the is the `permission.has()` based on?  the sender?

Reply:
Good catch. I meant to switch those to IDs. And the permissions are based on the sender. It is looking for these built in discord permissions:
![image](https://user-images.githubusercontent.com/6242524/55576888-ea472800-56d7-11e9-9cbc-79b9635db77c.png)

Reply:
Should we just drop the id/username check altogether then?  I think channel perms would be the more solid approach.

`isAdmin(permissions: ...)`

Reply:
Yeah, I think so. We can always revisit it later if needed